terracotta/bootstrap/src/main/java/net/sf/ehcache/terracotta/StandaloneTerracottaClusteredInstanceFactory.java

```diff
@@ -10,7 +10,7 @@
 public class StandaloneTerracottaClusteredInstanceFactory extends TerracottaClusteredInstanceFactory {
 
   public StandaloneTerracottaClusteredInstanceFactory(final TerracottaClientConfiguration terracottaConfig,
-                                                      ClassLoader loader) {
-    super(terracottaConfig, loader);
+                                                      String cacheManagerName, ClassLoader loader) {
```

Comment:
I'm wondering if this should be explicitly embedded inside the `TerracottaClientConfiguration`, rather than sitting alongside it?

Reply:
I thought of that but I didn't do it as it is used by actual users provide the terracotta client configuration while creating configuration for the whole cachemanager  something like - 
Configuration managerConfiguration2 = new Configuration()
                    .name("bigmemory-config" + counter.getAndIncrement())
                    .terracotta(new TerracottaClientConfiguration().url("localhost:9510").rejoin(false))
                    .cache(new CacheConfiguration()
                            .name("bigMemory2")
                            .maxEntriesLocalHeap(50)
                            .maxBytesLocalOffHeap(128, MemoryUnit.MEGABYTES)
                            .copyOnRead(true)
                            .eternal(true)
                            .terracotta(new TerracottaConfiguration())